packages/jellyfish-wallet-mnemonic/src/mnemonic.ts

```diff
@@ -151,26 +167,35 @@ export class MnemonicHdNode implements WalletHdNode {
   }
 }
 
+export interface SigningInterface {
+  unsigned: (buffer: Buffer, tx: Transaction) => Promise<void>
+  signed: (buffer: Buffer, tx: TransactionSegWit) => Promise<void>
```

Comment:
```suggestion
  after: (buffer: Buffer, tx: TransactionSegWit) => Promise<void>
```

Document this interface